### DISCUSSION TITLE
Question about symbols

### DISCUSSION THREAD

**Discussion by connorferster** [robbievanleeuwen/concrete-properties#5] [Q&A] [answered]
Hi Robbie,

I have some questions about the symbols used in AS3600 that are incorporated into `concrete-properties`:

1. In the `Concrete` material, we specify `alpha_1`. In the `WhitneyStressBlock`, we specify `alpha_2`. Are these essentially the same `alpha` given as `1 - 0.003 * fc'` (0.85 >= alpha >= 0.72)?
2. In the results, what does `mv` represent?

Thanks!

**Comment by robbievanleeuwen** [comment, marked as answer]
Hi @connorferster, I'm looking forward to getting the docs done so this all makes sense and is clear! Also looking forward to some fruitful discussions with an international community to see how ``Concrete`` needs to work for other codes! In answer to your questions:

**Question 1:**
``alpha_2`` relates to the confinement in the rectangular stress block during bending (this has been updated in the most recent code so the values in the examples are not correct):
<img src="https://user-images.githubusercontent.com/18841910/177566156-c5caf2b2-4424-49f3-aa72-11c9f7f8a415.png" width="300">

``alpha_1`` relates to confinement under a squash load (pure compression):
<img src="https://user-images.githubusercontent.com/18841910/177567317-7fc46123-2ada-49f6-a1df-641a4834dfc5.png" width="300">

**Question 2:**
``mv`` represents the moment about the local axis (``theta`` dependent). This should actually be ``m_u`` as I have chosen a ``u-v`` axis to be a local axis defined by ``theta``. I'll fix this up and hopefully make this all clear in the docs!

**Reply by connorferster** [reply]
Thank you!

The alpha_2 is the same as "alpha_1" in the Canadian code and the alpha_1 for squash load is represented as a "hard-coded" coefficient of 0.8 in the Canadian code.

I am going to make use of concrete-properties in my course starting in September. For the "final project", the engineers will combine it with pre-exported analysis results they will need to post-process in pandas. It is going to be great!

**Reply by robbievanleeuwen** [reply]
Can't wait to see how you incorporate it! Also some good motivation to wrap things up with the initial release of ``concrete-properties``!

**Reply by robbievanleeuwen** [reply]
@connorferster just a heads up that these symbols have now changed to make it a bit more 'code agnostic'.

Almost finished the docs, but plenty of info on materials now [here](https://robbievanleeuwen.github.io/concrete-properties/rst/materials.html).

**Reply by connorferster** [reply]
Such excellent documentation! A pleasure to read. Have been reading it on my lunch break. This is amazing.
